WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -146,7 +147,14 @@ class GutenbergViewController: UIViewController, PostEditor {
 
     // MARK: - Private variables
 
-    private lazy var gutenberg = Gutenberg(dataSource: self)
+    private lazy var gutenbergImageLoader: GutenbergImageLoader = {
+        return GutenbergImageLoader(post: self.post)
```

Comment:
```suggestion
        return GutenbergImageLoader(post: post)
```

No need for `self.` in this initializer